torch/nn/parameter.py

```diff
@@ -67,6 +69,9 @@ def __reduce_ex__(self, proto):
             (self.data, self.requires_grad, OrderedDict())
         )
 
+    def new_empty(self, shape):
+        return type(self)(super().new_empty(shape))
+
```

Comment:
@albanD to check -- this is supposed to have the same API as Tensor.new_empty, right? (we should expect Parameter, which is a subclass of torch.Tensor, to have the same API for a the method).

Tensor.new_empty takes in more arguments than just a shape, so I'll update it to have those if that sounds alright

Reply:
Yes we can do that. the simplest is to take `*args, **kwargs` and just forward them.